app/lib/controlled_work_mappings/cw1.yml

```diff
@@ -0,0 +1,495 @@
+
+# Dependents' allowances / Aged 15 or under
+- name: undefined_60
+  type: text
+  source: from_attribute
+  attribute: client_and_partner_under_16_dependant_allowance
+  skip_unless: client_income_relevant?
+
+# Dependents' allowances / Aged 16 or over
+- name: undefined_61
+  type: text
+  source: from_attribute
+  attribute: client_and_partner_16_plus_dependant_allowance
+  skip_unless: client_income_relevant?
+
+# Tax and national insurance / Client
+- name: undefined_62
+  type: text
+  source: from_attribute
+  attribute: client_tax_and_national_insurance
+  skip_unless: client_income_relevant?
+
+# Tax and national insurance / Partner
+- name: undefined_63
+  type: text
+  source: from_attribute
+  attribute: partner_tax_and_national_insurance
+  skip_unless: partner_income_relevant?
+
+# Standard allowance for employment expense / Client
+- name: undefined_64
+  type: text
+  source: from_attribute
+  attribute: client_employment_deduction
+  skip_unless: client_income_relevant?
+
+# Standard allowance for employment expense / Partner
+- name: undefined_65
+  type: text
+  source: from_attribute
+  attribute: partner_employment_deduction
+  skip_unless: partner_income_relevant?
+
+# Maintenance allowance / Client
+- name: undefined_66
+  type: text
+  source: from_cfe_payload
+  cfe_payload_location: result_summary.disposable_income.maintenance_allowance
+  skip_unless: client_income_relevant?
+
+# Maintenance allowance / Partner
+- name: undefined_67
+  type: text
+  source: from_cfe_payload
+  cfe_payload_location: result_summary.partner_disposable_income.maintenance_allowance
+  skip_unless: partner_income_relevant?
+
+# Childcare costs
+- name: undefined_68
+  type: text
+  source: from_attribute
+  attribute: combined_childcare_costs
+  skip_unless: client_income_relevant?
+
+
+# Payment of income contribution order / Client
+- name: undefined_69
+  type: text
+  source: from_cfe_payload
+  cfe_payload_location: assessment.disposable_income.monthly_equivalents.all_sources.legal_aid
+  skip_unless: client_income_relevant?
+
+# Payment of income contribution order / Partner
+- name: undefined_70
+  type: text
+  source: from_cfe_payload
+  cfe_payload_location: assessment.partner_disposable_income.monthly_equivalents.all_sources.legal_aid
+  skip_unless: partner_income_relevant?
+
+# Total allowances / Client
+- name: undefined_71
+  type: text
+  source: from_cfe_payload
+  cfe_payload_location: result_summary.disposable_income.total_outgoings_and_allowances
+  skip_unless: client_income_relevant?
+
+# Total allowances / Partner
+- name: undefined_72
+  type: text
+  source: from_cfe_payload
+  cfe_payload_location: result_summary.partner_disposable_income.total_outgoings_and_allowances
+  skip_unless: partner_income_relevant?
+
+# Total monthly disposable income / Client
+- name: undefined_73
+  type: text
+  source: from_cfe_payload
+  cfe_payload_location: result_summary.disposable_income.total_disposable_income
```

Comment:
just had a thought on this value based on some user research I was just in - it came up that minus values are not allowed on the CW forms (sounds obvious I know)

It's possible that these values can be a minus if the outgoings are somehow greater than the income. I wonder if we need to protect against that? Maybe we need a method here, so if the value is negative, return 0? What do you think?